ooniprobe/Test/Suite/ExperimentalSuite.m

```diff
@@ -11,12 +11,15 @@ -(id) init {
     return self;
 }
 
-- (NSArray*)getTestList {
-    if ([self.testList count] == 0){
-        [self.testList addObject:[[Experimental alloc] initWithName:@"stunreachability"]];
-        [self.testList addObject:[[Experimental alloc] initWithName:@"dnscheck"]];
-        [self.testList addObject:[[Experimental alloc] initWithName:@"torsf"]];
-        [self.testList addObject:[[Experimental alloc] initWithName:@"vanilla_tor"]];
+- (NSArray *)getTestList {
+    if ([self.testList count] == 0) {
+        if (self.autoRUn) {
+            [self.testList addObject:[[Experimental alloc] initWithName:@"torsf"]];
+            [self.testList addObject:[[Experimental alloc] initWithName:@"vanilla_tor"]];
+        } else {
+            [self.testList addObject:[[Experimental alloc] initWithName:@"stunreachability"]];
+            [self.testList addObject:[[Experimental alloc] initWithName:@"dnscheck"]];
+        }
```

Comment:
```suggestion
        if (self.autoRUn) {
            [self.testList addObject:[[Experimental alloc] initWithName:@"torsf"]];
            [self.testList addObject:[[Experimental alloc] initWithName:@"vanilla_tor"]];
        }
        [self.testList addObject:[[Experimental alloc] initWithName:@"stunreachability"]];
        [self.testList addObject:[[Experimental alloc] initWithName:@"dnscheck"]];
```

We should run all experiments in auto-run mode and just the fast ones otherwise